server/db/index.js

```diff
@@ -0,0 +1,19 @@
+const mysql = require('mysql');
+const createTables = require('./config');
+const Promise = require('bluebird');
```

Comment:
You may want to install bluebird as a dependency in package.json

Reply:
Or add bluebird to the readme as something to npm install

Reply:
good call. submitting a new PR with bluebird as a dependency